### PR TITLE
Add apidef files for Atlas

### PR DESCRIPTION
**Pull request by Colin-Stone**
Signed-off-by: Colin-Stone <30794003+Colin-Stone@users.noreply.github.com>
Merged in zosmf code too

### CODE REVIEW

File: scripts/zowe-api-mediation-configure.sh [hunk 1]
```diff
@@ -78,6 +78,102 @@ cd ..
...
+          serviceRelativeUrl:
+EOF
+iconv -f IBM-1047 -t IBM-850 $TEMP_DIR/zosmf.yml > $STATIC_DEF_CONFIG/zosmf.yml	
+
+# Add static definition for MVS datasets
+cat <<EOF >$TEMP_DIR/datasets.yml
+# Static definition for MVS Datasets
+#
+# Once configured you can access MVS Datasets via the API gateway:
+# http --verify=no GET https://$ZOWE_EXPLORER_HOST:$ZOWE_EXPLORER_SERVER_HTTPS_PORT/api/v1/datasets{HLQ} 'X-CSRF-ZOSMF-HEADER;'
```
Comment (stevenhorsman) [inline scripts/zowe-api-mediation-configure.sh]: Don't add the zosmf header

File: scripts/zowe-api-mediation-configure.sh [hunk 2]
```diff
@@ -78,6 +78,102 @@ cd ..
...
+          serviceRelativeUrl: ui/v1/datasets
+EOF
+iconv -f IBM-1047 -t IBM-850 $TEMP_DIR/datasets.yml > $STATIC_DEF_CONFIG/datasets.yml	
+
+# Add static definition for Jobs
+cat <<EOF >$TEMP_DIR/jobs.yml
+# Static definition for Jobs
+#
+# Once configured you can access Jobs via the API gateway:
+# http --verify=no GET https://$ZOWE_EXPLORER_HOST:$ZOWE_EXPLORER_SERVER_HTTPS_PORT/api/v1/jobs?prefix=*' 'X-CSRF-ZOSMF-HEADER;'
```
Comment (stevenhorsman) [inline scripts/zowe-api-mediation-configure.sh]: Same as above

File: scripts/zowe-api-mediation-configure.sh [hunk 3]
```diff
@@ -78,6 +78,102 @@ cd ..
...
+          serviceRelativeUrl: ui/v1/jobs
+EOF
+iconv -f IBM-1047 -t IBM-850 $TEMP_DIR/jobs.yml > $STATIC_DEF_CONFIG/jobs.yml	
+
+# Add static definition for zos
+cat <<EOF >$TEMP_DIR/zos.yml
+# Static definition for zOS
+#
+# Once configured you can access Jobs via the API gateway:
+# http --verify=no GET https://$ZOWE_EXPLORER_HOST:$ZOWE_EXPLORER_SERVER_HTTPS_PORT/api/v1/zos/username' 'X-CSRF-ZOSMF-HEADER;'
```
Comment (stevenhorsman) [inline scripts/zowe-api-mediation-configure.sh]: Ditto

File: scripts/zowe-api-mediation-configure.sh [hunk 4]
```diff
@@ -78,6 +78,102 @@ cd ..
...
+
+EOF
+iconv -f IBM-1047 -t IBM-850 $TEMP_DIR/zos.yml > $STATIC_DEF_CONFIG/zos.yml	
+
+# Add static definition for languages
+cat <<EOF >$TEMP_DIR/orion.yml
+# Static definition for Orion Languages
+#
+# Once configured you can access orion languages via the API gateway:
+# http --verify=no GET https://$ZOWE_EXPLORER_HOST:$ZOWE_EXPLORER_SERVER_HTTPS_PORT/explorer-languages/orion/languages.json' 'X-CSRF-ZOSMF-HEADER;'
```
Comment (stevenhorsman) [inline scripts/zowe-api-mediation-configure.sh]: Add again.

Comment (Colin-Stone) [reply scripts/zowe-api-mediation-configure.sh]: Whipped them out

**Pull request merged**